src/_includes/banner.html

```diff
@@ -2,7 +2,7 @@
   For headings use h3 elements.
 {% endcomment -%}
 <div class="site-banner site-banner--default" role="alert">
-  <a href="https://medium.com/flutter/flutter-3-24-dart-3-5-204b7d20c45d">Flutter 3.24 and Dart 3.5</a> are here!
-  Check out <a href="/release/whats-new">what's new</a> on the website.
+  Flutter 3.27 is here to celebrate Flutter's production era!
```

Comment:
I wonder if we want to leave out the 3.7 part (which is relatively minor in the overall messaging), and just have something like "Celebrating Flutter's production era! [link]" ?

Reply:
Sounds good and makes sense to me :)

```suggestion
  Celebrating Flutter's production era!
```